DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Northern\MaintenanceModeBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder('northern_maintenance_mode');
+
+        $treeBuilder->getRootNode()
+                    ->children()
+                    ->scalarNode('maintenance_flag_path')
```

Comment:
One note - the way that this is set up, it would be nicer to nest these, but I don't know if that's possible to set up with PHPStorm formatting.

Reply:
If you mean indentation, you should be able to use comments to disable and then enable the formatter around this chunk of code

Reply:
Ah yeah, I did try this - it just wasn't enabled in PHPStorm. I also added an `.editorconfig` file and ensured that the formatting enable/disable tags are in there.